charts/shoot-addons/charts/kubernetes-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -25,6 +25,8 @@ spec:
         origin: gardener
         k8s-app: kubernetes-dashboard
     spec:
+      nodeSelector:
+        worker.gardener.cloud/system-components: true
```

Comment:
The value of the selector must be escaped 
```suggestion
        worker.gardener.cloud/system-components: "true"
```

If you try to apply this manifest, on minukube for example or any other k8s cluster,
```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: vpn-shoot
  name: vpn-shoot
  namespace: kube-system
spec:
  replicas: 1
  revisionHistoryLimit: 0
  selector:
    matchLabels:
      app: vpn-shoot
  strategy:
    rollingUpdate:
      maxSurge: 25%
      maxUnavailable: 25%
    type: RollingUpdate
  template:
    metadata:
      labels:
        app: vpn-shoot
    spec:
      containers:
      - image: eu.gcr.io/gardener-project/gardener/vpn-shoot:0.19.0
        imagePullPolicy: IfNotPresent
        name: vpn-shoot
      nodeSelector:
        worker.gardener.cloud/system-components: true
      restartPolicy: Always

```

you will get this error
```
Error from server (BadRequest): error when creating "vpn.yaml": Deployment in version "v1" cannot be handled as a Deployment: v1.Deployment.Spec: v1.DeploymentSpec.Template: v1.PodTemplateSpec.Spec: v1.PodSpec.NodeSelector: ReadString: expects " or n, but found t, error found in #10 byte of ...|ponents":true},"rest|..., bigger context ...|ctor":{"worker.gardener.cloud/system-components":true},"restartPolicy":"Always"}}}}

```

Reply:
Fixed